enterprise/frontend/src/metabase-enterprise/rich_text_editing/tiptap/extensions/Command/CommandSuggestion.tsx

```diff
@@ -109,76 +100,18 @@ export const CommandSuggestion = forwardRef<
 >(function CommandSuggestionComponent({ command, editor, query }, ref) {
   const document = useSelector(getCurrentDocument);
   const [selectedIndex, setSelectedIndex] = useState(0);
-  const [showLinkSearch, setShowLinkSearch] = useState(false);
```

Comment:
Had to refactor this binary flags (`showLinkSearch`, `showEmbedSearch`) which are actually mutually exclusive to a single state - `viewMode` .